meson.build

```diff
@@ -39,13 +39,15 @@ cc_dict = {
     'compiler': meson.get_compiler('c'),
     'machine': host_machine,
     'config': conf,
```

Comment:
`conf` and `conf_native` are no longer used. Please delete them from the lines 35 and 36.
Also it appears that the `config` key in the dictionary is not needed and you can just create an empty `configuration_data()` object in the loop instead of getting an empty version from the dict.

What do you think?

Reply:
Roger